src/embed_tests/TestFinalizer.cs

```diff
@@ -104,33 +88,45 @@ public void CollectBasicObject()
         }
 
         [Test]
-        [Ignore("Ignore temporarily")]
+        [Obsolete("GC tests are not guaranteed")]
```

Comment:
Is NUnit aware of this attribute or does this just generate yet another compile-time warning? Does it help to retry the test using https://docs.nunit.org/articles/nunit/writing-tests/attributes/retry.html?

Reply:
NUnit is not aware. This actually removes warning from the usage of other `[Obsolete]` code.